lib/03_simon_says.rb

```diff
@@ -0,0 +1,26 @@
+def echo(word)
+  word
+end
+
+def shout(word)
+  word.upcase
+end
+
+def repeat(word, times = 2)
+  ((word + " ") * times).strip
+end
+
+def start_of_word(word, num)
+  word[0..num - 1]
+end
+
+def first_word(phrase)
+  phrase.split.first
+end
+
+def titleize(phrase)
+  little_words = %w(and over the a)
+  arr = phrase.split
+  arr.each { |word| word[0] = word[0].upcase unless little_words.include?(word) && word != arr.first}
```

Comment:
For readability - since this runs over about 80 characters, it's better to write this as a block: 

```ruby
.each do |word|
  ...
end
```

instead of putting it in a block inside of braces.